stdlib/asm/math/u64.masm

```diff
@@ -451,7 +451,7 @@ export.unchecked_div
     eq.0
     assert
 
-    adv_push.2          # read the remainder from the advice tape and make sure it consists of
+    adv_push.2          # read the remainder from the advice stack and make sure it consists of
```

Comment:
Let's change this to: "pop the remainder from the advice stack and make sure it consists of"

Let's also make similar changes in the rest of this file.